src/sentry/models/useremail.py

```diff
@@ -80,3 +83,15 @@ def is_primary(self):
     def get_primary_email(cls, user: User) -> UserEmail:
         """@deprecated"""
         return cls.objects.get_primary_email(user)
+
+    # TODO(getsentry/team-ospo#181): what's up with email/useremail here? Seems like both get added
+    # with `sentry.user` simultaneously? Will need to make more robust user handling logic, and to
+    # test what happens when a UserEmail already exists.
```

Comment:
UserEmails are required to be unique per user. Duplicate emails between users are ok and expected.

Reply:
Fixed this in a follow-on PR, since it made more sense to validate it once the correct test suite was written (after this PR in the stack).

Reply:
https://github.com/getsentry/sentry/pull/55433 is the fix. Okay to leave this as is until then?